internal/plugin.go

```diff
@@ -18,7 +18,9 @@ import (
 
 const (
 	configPolicyKind           = "ConfigurationPolicy"
-	policyAPIVersion           = "policy.open-cluster-management.io/v1"
+	policyAPIGroup             = "policy.open-cluster-management.io"
+	v1                         = "v1"
+	v1beta1                    = "v1beta1"
```

Comment:
I'm not sure I see the benefit of naming the variable names the same as their content. Would it make more sense to call them `policyAPIVersion` and `policySetAPIVersion`? And then we can change the content without needing to update the variable name.

Reply:
This would save us two split calls on line 1147 and 1156

Reply:
Let me update. I can make it variable too